### DISCUSSION TITLE
Display Paths of Client Certs in Workspace Settings -> Client Certificates

### DISCUSSION THREAD

**Discussion by Vyoam** [Kong/insomnia#3538] [Ideas] [answered]
**Is your feature request related to a problem? Please describe.**
When dealing with multiple client certs, it'd be good to conveniently look up the paths where the client certs are stored, especially as the edit feature is absent.

<img width="576" alt="T" src="https://user-images.githubusercontent.com/1176140/84770676-9af7c980-aff5-11ea-9e39-97e8b8b7ab38.png">

**Describe the solution you'd like**
The straightforward approach would be to just add a bunch of rows (1 for pfx and/or 2 for key-crt) for each host entry, displaying the relevant paths.

Code-wise, seems this would need change: https://github.com/Kong/insomnia/blob/0a616fba6b12e3e51e3643f5f85f9fd27471f762/packages/insomnia-app/app/ui/components/modals/workspace-settings-modal.js ... renderCertificate(certificate: ClientCertificate)

**Describe alternatives you've considered**
For now, I can look up the data in %appdata%\Insomnia\insomnia.ClientCertificate.db

**Comment by dimitropoulos** [comment, marked as answer]
There's a PR up for this https://github.com/Kong/insomnia/pull/2714

**Reply by develohpanda** [reply]
This has now been merged
